### ISSUE TITLE
0.3.1 doesn't appear in 'apt show git-secret' using https://dl.bintray.com/sobolevn/deb

### ISSUE THREAD

**Issue by dkrutsko** [sobolevn/git-secret#460]
I'm looking to set up a local clone of your Debian repository but it looks like the latest version (`0.2.5`) is not properly published. The `*.deb` file is [there](https://dl.bintray.com/sobolevn/deb) but not actually exposed. The RPM repository, on the other hand, is correct.

**Comment by joshrabinowitz** [comment]
Hello @dkrutsko 

Thank you for your feedback.

I don't completely understand what you're asking, probably partly because I'm not a Debian expert.

Which repository are you trying to clone? 
And can you please detail what should be changed in git-secret to fix this issue?

**Comment by dkrutsko** [comment]
Sure, I'm looking at the installation instructions found [here](https://git-secret.io/installation). It says to add `https://dl.bintray.com/sobolevn/deb` to the list of sources, however, that repository doesn't appear to include `0.2.5`. For example, when I install `git-secret` using `sudo apt-get install git-secret`, it only installs `0.2.4` on Ubuntu 16. Doing a `curl` (seel below) on the packages list, it appears that `0.2.5` isn't listed, despite having the `*.deb` file in the repository. Could it be that you need to regenerate the repository metadata? or am I doing something wrong. I am using [`apt-mirror`](https://apt-mirror.github.io) to clone this repository.

`curl https://dl.bintray.com/sobolevn/deb/dists/git-secret/main/binary-all/Packages`

**Comment by joshrabinowitz** [comment]
I see what you're talking about, @dkrutsko. I'll try to look into this soon.

**Comment by joshrabinowitz** [comment]
Hi @dkrutsko I'm pretty sure this is because we haven't officially released 0.2.5 yet.

**Comment by joshrabinowitz** [comment]
Hey @dkrutsko has this been resolved with the release of 0.2.6?

**Comment by dkrutsko** [comment]
Hello, thanks for looking into this. I just checked and it looks like `0.2.5` is there but not `0.2.6`.

```bash
curl -sS https://api.bintray.com/users/sobolevn/keys/gpg/public.key | sudo apt-key add -
printf -- "deb https://dl.bintray.com/sobolevn/deb git-secret main" | sudo tee /etc/apt/sources.list.d/git-secret.list

sudo apt -y update
sudo apt -y install git-secret

git-secret --version
# Prints 0.2.5

apt-cache show git-secret | grep Version
# Prints
# Version: 0.2.5
# Version: 0.2.4
# Version: 0.2.2
# Version: 0.2.1
```

**Comment by joshrabinowitz** [comment]
Thanks for getting back to me. @simbo1905 do you know what's going on?

**Comment by artsiom-streetshares** [comment]
Hey guys, any solution for the issue? I really need to install 0.2.6.

**Comment by artsiom-streetshares** [comment]
@dkrutsko I'm getting 404 for that one, and for everything in that repo in fact

**Comment by dkrutsko** [comment]
@artsiom-streetshares My apologies, seems the link had an extra colon in it, I edited it, could you try again?

**Comment by artsiom-streetshares** [comment]
@dkrutsko thanks, now it works!

**Comment by dkrutsko** [comment]
Nice, is it me or does `0.3.1` not show up now? Seems like apt is always a version behind?

**Comment by joshrabinowitz** [comment]
@dkrutsko I see 0.3.1 at https://dl.bintray.com/sobolevn/deb/ 
and I see 0.3.1 at 
https://bintray.com/sobolevn/deb/git-secret/0.3.1

Where is it not showing up?

**Comment by dkrutsko** [comment]
See my comment here: https://github.com/sobolevn/git-secret/issues/460#issuecomment-500426441

**Comment by joshrabinowitz** [comment]
@dkrutsko I see what you're talking about now. 

I followed your steps in https://github.com/sobolevn/git-secret/issues/460#issuecomment-500426441 and I found i installed 0.3.0 also, even though 0.3.1 is at https://dl.bintray.com/sobolevn/deb/

I have no idea what's going on here, though I do note that the [Packages file](https://dl.bintray.com/sobolevn/deb/dists/git-secret/main/binary-all/Packages) doesn't list 0.3.1.

@simbo1905 / @sobolevn any idea what's up here?

**Comment by joshrabinowitz** [comment]
#312 could be related to this

**Comment by joshrabinowitz** [comment]
Closed as dup, see cleaned up issue #551
